assets/js/theme/common/product-details-base.js

```diff
@@ -145,8 +145,14 @@ export default class ProductDetailsBase {
      */
     getViewModel($scope) {
         return {
-            $priceWithTax: $('[data-product-price-with-tax]', $scope),
-            $priceWithoutTax: $('[data-product-price-without-tax]', $scope),
+            priceWithTax: {
+                $div: $('.price--withTax', $scope),
+                $span: $('[data-product-price-with-tax]', $scope),
```

Comment:
💅 I would change class to data attribute selector to be consistent  
`$div: $('.price--withTax', $scope)` ➡️ `$div: $('[data-price-with-tax-container]', $scope)`

Reply:
I set these selectors to be consistent with the rest of the prices ​​and properties in the getViewModel method, for example:
```
rrpWithTax: {
    $div: $('.rrp-price--withTax', $scope),
    $span: $('[data-product-rrp-with-tax]', $scope),
},
rrpWithoutTax: {
    $div: $('.rrp-price--withoutTax', $scope),
    $span: $('[data-product-rrp-price-without-tax]', $scope),
},
```